app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
```

Comment:
A comment belongs to an `author` as you use a `custom foreign key` named `author_id` in comments table.
`For example: belongs_to :author, class_name: 'User'`